libobs/obs.h

```diff
@@ -291,6 +297,16 @@ EXPORT uint32_t obs_get_version(void);
 /** @return The current core version string */
 EXPORT const char *obs_get_version_string(void);
 
+/** Sets things up for calls to obs_get_cmdline_args. Called once at startup. */
```

Comment:
Please document the function parameters.

Reply:
Sure, will do.